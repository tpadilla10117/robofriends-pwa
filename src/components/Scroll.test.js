import { shallow, mount, render } from 'enzyme';
import React from 'react';
import Scroll from './Scroll';

describe('Scroll.js coponent:', () => {
    it('renders the Scroll.js component', () => {
        expect.assertions(2);
        expect(shallow(<Scroll/>)).toMatchSnapshot();
        expect(shallow(<Scroll/>).length).toEqual(1)
    })
})
