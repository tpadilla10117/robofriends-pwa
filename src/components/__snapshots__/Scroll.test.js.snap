// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll.js coponent: renders the Scroll.js component 1`] = `ShallowWrapper {}`;
